app/models/config.rb

```diff
@@ -1,6 +1,6 @@
 class Config < ActiveRecord::Base
   belongs_to :project
-  scope :for, ->(metric_name) { where(:metric_name => metric_name).limit(1) }
+  scope :for, ->(metric_name) { where(:metric_name => metric_name).first }
```

Comment:
yeah, I wonder if rails scopes are sophisticated enough to produce the same optimal SQL ...? Another issue with this kind of code is whether the naming is sufficient to indicate what happens under the hood ...